src/splitInteger.test.js

```diff
@@ -4,18 +4,22 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
```

Comment:
Add a more comprehensive test suite. Consider edge cases, such as value = 0, numberOfParts = 0, and negative inputs 🧪